src/home-page/components/DemoChart.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+
+export const DemoChart: React.FC<{}> = () => (
+  <div>
+    <Bar
+      data={{
```

Comment:
I think it could be better to read, if we declare the data and options on the top and use it here for example like this `data={data}`

Reply:
Good idea. It's done.